vertexai/src/GenerativeModel.cs

```diff
@@ -148,17 +142,20 @@ public Chat StartChat(IEnumerable<ModelContent> history) {
   }
 #endregion
 
+  private void SetRequestHeaders(HttpRequestMessage request) {
+    request.Headers.Add("x-goog-api-key", _firebaseApp.Options.ApiKey);
+    request.Headers.Add("x-goog-api-client", "genai-csharp/0.1.0");
```

Comment:
Is this fixed number?

Reply:
Good catch, it shouldn't be.  Getting the actual version number will take some additional changes to App to expose it, so I'm going to mark it with a TODO for now.